problem_builder/questionnaire.py

```diff
@@ -158,14 +158,12 @@ def author_edit_view(self, context):
         """
         Add some HTML to the author view that allows authors to add choices and tips.
         """
+        from .mentoring import MentoringWithExplicitStepsBlock
+
         fragment = self.get_author_edit_view_fragment(context)
 
-        # Let the parent block determine whether to display buttons to add review-related child blocks.
-        # * Problem Builder units use MentoringBlock parent components, which define an 'is_assessment' property,
-        #   indicating whether the (deprecated) assessment mode is enabled.
-        # * Step Builder units can show review components in the Review Step.
         fragment.add_content(loader.render_template('templates/html/questionnaire_add_buttons.html', {
```

Comment:
@symbolist could you add some comment here, I see you removed the previous one, cause it had some mention of assessment, a part of it is not valid anymore, but the steps section still is.

Reply:
@rocioar Sure. The comment was just an explanation for `getattr(self.get_parent(), 'is_assessment', True)` and I thought its replacement was evident enough to not require a comment.